components/content_settings/renderer/brave_content_settings_agent_impl.cc

```diff
@@ -344,6 +346,16 @@ BraveFarblingLevel BraveContentSettingsAgentImpl::GetBraveFarblingLevel() {
     }
   }
 
+  if (!webcompat_features_read_) {
+    GetOrCreateBraveShieldsRemote()->GetWebcompatExceptions(
+        GetOriginOrURL(frame), &webcompat_features_);
+    webcompat_features_read_ = true;
```

Comment:
webcompat_features_ could be optional instead

Reply:
This code has been removed.